src/TimeSteppers/clock.jl

```diff
@@ -36,6 +36,8 @@ next_time(clock::Clock{<:AbstractTime}, Δt) = clock.time + Nanosecond(round(Int
 tick_time!(clock, Δt) = clock.time += Δt
 tick_time!(clock::Clock{<:AbstractTime}, Δt) = clock.time += Nanosecond(round(Int, 1e9 * Δt))
 
+Time(clock::Clock) = Time(clock.time)

```

Comment:
do we need to import `Time`?

Reply:
no we should export it

Reply:
actually it is exported

Reply:
yea but to extent the constructor `Time`, we need to import it here since it's defined in `Units`